data/blog/electron-5-0.md

```diff
@@ -0,0 +1,115 @@
+
+In addition, Electron 5 includes changes to Electron-specific APIs. You can find a summary of the major changes in Electron 5 below; for the full list of changes, check out the [Electron v5.0.0 release notes](https://github.com/electron/electron/releases/tag/v5.0.0).
+
+### Promisification
+
+Electron 5 contains changes as part of our [Promisification](https://github.com/electron/electron/blob/master/docs/api/modernization/promisification.md) initiative to convert callback-based functions in Electron to return Promises.  The following APIs were converted for Electron
+5:
+* `app.getFileIcon`
+* `contentTracing.getCategories`
+* `contentTracing.startRecording`
+* `contentTracing.stopRecording`
+* `debugger.sendCommand`
+* Cookies API
+* `shell.openExternal`
+* `webContents.loadFile`
+* `webContents.loadURL`
+* `webContents.zoomLevel`
+* `webContents.zoomFactor`
+* `win.capturePage`
+
+### System colors access for macOS
+
+Three functions were changes or added to `systemPreferences` to access macOS systems
+colors.  Those functions are:
+* `systemPreferences.getAccentColor`
+* `systemPreferences.getColor`
+* `systemPreferences.getSystemColor`
+
+### Process memory information
+
+The function `process.getProcessMemoryInfo` has been added to get memory usage
+statistics about the current process.
+
+### Additional filtering for remote APIs.
+
+In an effort to improve security but still allow use of the remote API, there are now
+new remote events so that `remote.getBuiltin`, `remote.getCurrentWindow`, `remote.getCurrentWebContents` and `<webview>.getWebContents` can be filtered.
+
+### Multiple BrowserViews on BrowserWindow
+
+BrowserWindow now supports managing multiple BrowserViews within the same BrowserWindow.
+
+## Breaking Changes
+
+### Defaults for packaged apps
+
+Packaged apps will now behave the same as the default app, which means that a default
+application menu will be created unless the app has one and the `window-all-closed`
+event will be automatically handled unless the app handles the event.
+
+### Mixed sandbox
+
+Mixed sandbox mode is now enabled by default.  Renderers launched with `sandbox: true`
+will now be actually sandboxed, where previously they would only be sandboxed if mixed-sandbox mode was also enabled.
+
+### Security improvements
+The default values of `nodeIntegration` and `webviewTag` are now false to improve security.
+
+### Spellchecker now asynchronous
+
+The SpellCheck API has been changed to provide asynchronous results.
+
+## Deprecations
+
+The following breaking changes are planned for Electron 6.0, and thus are deprecated in Electron 5.0.
+
+### Mksnapshot binaries for arm and arm64
+Native binaries of mksnapshot for arm and arm64 are deprecated and will be removed in 6.0.0.  Snapshots can be created for arm and arm64 using the x64 binaries.
+
+### ServiceWorker APIs on WebContents
+Deprecated ServiceWorker APIs on WebContents in preparation for their removal.
```

Comment:
Can we list them here, people shouldn't have to google "electron service worker APIs", this currently just raises more questions

Reply:
Fixed 12f833c310d21e7525ebb3556fe833ed343594de